### ISSUE TITLE
Tubulonodular Pericallosal Lipoma

### ISSUE THREAD

**Issue by taylordm** [obophenotype/human-phenotype-ontology#7097]
Preferred term label:

Tubulonodular Pericallosal Lipoma

Synonyms

Definition (free text, please give PubMed ID)
 
Tubulonodular pericallosal lipomas are the more common variety of pericallosal lipomas. They are rounded or lobular and usually measure >2 cm in thickness. They are anteriorly situated and are associated with extensive callosal and often fronto-facial anomalies. The tubulonodular variety can extend into the choroid plexus / lateral ventricles. 
Adapted from https://radiopaedia.org/articles/pericallosal-lipoma?lang=us

References: 
Curvilinear and tubulonodular varieties of lipoma of the corpus callosum: an MR and CT study.  PMID: 1885799

Prenatal diagnosis of pericallosal lipoma: Systematic review PMID: 31587959

Parent term (use hpo.jax.org/app)
Parent term will be "Pericallosal Lipoma" submitted as a separate issue term request.

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
Pai disease ORPHA:1993 is one syndrome that has these lipomas but not the only manifestation as they can occur on their own without any other symptoms.

Your nano-attribution (ORCID)
0000-0002-3302-4610

**Comment by pnrobinson** [comment]
thanks @taylordm added term as HP_0034014
